nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -142,6 +166,19 @@ def log_accuracy_statistics(uncompressed_model_accuracy,
         nncf_logger.info('Relative accuracy drop: {:.2f}%'.format(rel_accuracy_drop))
         nncf_logger.info('Accuracy budget: {:.4f}'.format(accuracy_budget))
 
+    @property
+    def final_statistics(self):
+        statistics_dict = {
```

Comment:
Would you consider to reuse class based approach for the statistics form compression algorithms instead of dict?

Reply:
Done